kotlin-guice/src/main/kotlin/com/authzee/kotlinguice4/multibindings/KotlinMultibindingsScanner.kt

```diff
@@ -87,10 +84,8 @@ object KotlinMultibindingsScanner {
                 }
                 is ProvidesIntoOptional -> {
                     return when (annotation.value) {
-                        ProvidesIntoOptional.Type.DEFAULT -> KotlinOptionalBinder
-                                .newRealOptionalBinder(binder, key).getKeyForDefaultBinding()
-                        ProvidesIntoOptional.Type.ACTUAL -> KotlinOptionalBinder
-                                .newRealOptionalBinder(binder, key).getKeyForActualBinding()
+                        ProvidesIntoOptional.Type.DEFAULT -> KotlinOptionalBinder.newRealOptionalBinder(binder, key).getKeyForDefaultBinding()
```

Comment:
I was trying to keep line lengths below 100 characters, and it looks like here and in some other files the formatting was changed, removing some wrapping, and these lines are over 100 characters now.

Reply:
Sorry about that. I'll look into it when I get some free time.